src/decays/specializations/Ah/decay_Ah_to_lbarl.inc

```diff
@@ -33,5 +33,22 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, bar<ChargedLepton>::type,
       res *= 1. + get_alpha(context)/Pi*17./4.;
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.